t/apicast-policy-rate-limit.t

```diff
@@ -75,12 +75,10 @@ Return 200 code.
 
   location /flush_redis {
     content_by_lua_block {
-      local env = require('resty.env')
-      local redis_host = "$TEST_NGINX_REDIS_HOST" or '127.0.0.1'
-      local redis_port = "$TEST_NGINX_REDIS_PORT" or 6379
-      local redis = require('resty.redis'):new()
-      redis:connect(redis_host, redis_port)
-      redis:select(1)
+      local redis = require('apicast.threescale_utils').connect_redis({
+         host = "$TEST_NGINX_REDIS_HOST" or '127.0.0.1',     
```

Comment:
this can never work 😢 `"" or "whatever"` is always going to be `""`

Reply:
Fixed thanks!